src/main/resources/initEnv

```diff
@@ -46,3 +46,11 @@ fi
 set -a
 eval "$VARS"
 set +a
+
+not_sourced() {
+	test "X$(basename -- "$0")" = "XinitEnv"
+}
+if not_sourced; then
+	# when the script is invoked directly with parameters execute them in the bootstrapped context
+	$@
```

Comment:
Shouldn't this rather be `"$@"` to preserve quoted arguments?
E.g. if I want to run `./initEnv npm test --myarg="hello world"` it will pass `--myarg=hello` and a separate `world` arg to `npm test`

I always find this topic a bit hazy. For example I don't know if there are opposite edge cases where `$@` without quotes is necessary.

Reply:
You are right, without the quotes it will behave just like you described, and it's not good. I'll update the PR.